### PR TITLE
Fix crash when trying to get speculative info on too small an expression

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by CyrusNajmabadi** [comment]
Preemptively merging in.

File: src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs [hunk 1]
```diff
@@ -243,6 +265,36 @@ private static IEnumerable<SyntaxTrivia> MassageTrivia(IEnumerable<SyntaxTrivia>
...
+                    return semanticModel.TryGetSpeculativeSemanticModel(position, statement, out speculativeModel);
+                case EqualsValueClauseSyntax equalsValue:
+                    return semanticModel.TryGetSpeculativeSemanticModel(position, equalsValue, out speculativeModel);
+                case ArrowExpressionClauseSyntax arrowExpression:
+                    return semanticModel.TryGetSpeculativeSemanticModel(position, arrowExpression, out speculativeModel);
+                case ConstructorInitializerSyntax constructorInitializer:
+                    return semanticModel.TryGetSpeculativeSemanticModel(position, constructorInitializer, out speculativeModel);
+            }
+
+            speculativeModel = null;
```
Comment (jasonmalinowski) [inline src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs]: What does this have to do for C# script?

**Comment by jasonmalinowski** [comment]
@CyrusNajmabadi I admit I have no clue how this fixes it. Why wasn't this fixed in GetSpeculativeSymbolInfo? How do know not to write this bug again?

**Pull request merged**